lib/plugin/recma-jsx-rewrite.js

```diff
@@ -182,10 +179,7 @@ export function recmaJsxRewrite(options = {}) {
             }
 
             if (node.closingElement) {
-              node.closingElement.name = toJsxIdOrMemberExpression([
-                '_components',
-                id
-              ])
+              node.closingElement.name = node.openingElement.name
```

Comment:
We can’t reuse this object by reference. It has to be a new node unfortunately

Reply:
Sorry for my lack of understanding. Didn't think that would be problematic.

Reply:
No problem at all! 🙂